submissions/semenjob/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <ul class="nav__list">
+        <li tabindex="1" class="header__list">
+          <a class="head__link" href="#">Mail</a>
+        </li>
+        <li tabindex="1" class="header__list">
+          <a class="head__link" href="#">Images</a>
+        </li>
+      </ul>
+      <input
+        tabindex="1"
+        class="popup__checkbox"
```

Comment:
Is there any reason to name class and id with the same name? My suggestion that it can be confusing if you will use styles for `.popup__checkbox` and `#popup__checkbox`

Reply:
I agree)